### ISSUE TITLE
Custom domain SSL cert for SaaS (Medium) causes forced sign out

### ISSUE THREAD

**Issue by blainsmith** [brave/browser-laptop#2561]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
I can't tell if this is specific to Medium or not, but when visiting any Medium blog with a custom domain assigned to it the SSL cert seems to fail and Medium signs me out.

**Expected behavior:**
Not sign users out unless this is not Brave's issue and more of an issue with how Medium is assigning certs.

- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS

- Brave Version:
0.11.0

- Steps to reproduce:
    1. Visit oddcast.oddnetworks.com which is a Medium blog with a custom domain.
    2. See the alert drop down indicating you have been signed out.

- Screenshot if needed:
![brave-version](https://cloud.githubusercontent.com/assets/483829/16916194/9a6b04d4-4cc9-11e6-8a63-3dddd377e651.png)

![medium-brave](https://cloud.githubusercontent.com/assets/483829/16916204/a8d00a1a-4cc9-11e6-93c3-9459a27070a9.png)

![medium-chrome](https://cloud.githubusercontent.com/assets/483829/16916207/b00b0d02-4cc9-11e6-98e0-cb0c0a343f5b.png)

- Any related issues:
Selecting any of the domains under **Secure Origins** shows details of the certificate, but the **Open full certificate details** button does not do anything.

**Comment by DocWatson** [comment]
I am having the exact same issue on the same version of Brave and also on OSX. Interesting.

**Comment by diracdeltas** [comment]
Does the issue go away if you open the Bravery panel and select 'Allow all cookies'?

**Comment by blainsmith** [comment]
No, the issue persists. I even put "Shields Down" and it still happens.
